src/aks-preview/setup.cfg

```diff
@@ -0,0 +1,5 @@
+[bdist_wheel]
+universal=1
+
+[install]
+prefix=
```

Comment:
Remove the [install] section

Reply:
I am using pip[installed by homebrew] on mac to install this extension, got error if I don't add this snippet. https://stackoverflow.com/questions/24257803/distutilsoptionerror-must-supply-either-home-or-prefix-exec-prefix-not-both
But but release, it is ok to remove, since we only need .wheel package.